test/inhfd/memfd.py

```diff
@@ -0,0 +1,49 @@
+import os
+import ctypes
+import subprocess
+libc = ctypes.CDLL(None)
+
+
+# The SYS_memfd_create value depends on the architecture.
+# We don't have a good way to get the system call number.
+# Here, we invoke gcc. Oh well.
+def __get_SYS_memfd_create():
+    prog = """
+    #include <sys/syscall.h>
+    #include <stdio.h>
+    int main() {
+      printf("%d", SYS_memfd_create);
+      return 0;
+    }
+    """
+    cmd = "echo '{}' | gcc -x c - -o memfd_out && ./memfd_out && rm -f ./memfd_out".format(prog)
+    out = subprocess.check_output(cmd, shell=True)
+    return int(out)
+
+
+SYS_memfd_create = __get_SYS_memfd_create()
+
+
+def memfd_create(name, flags):
+    return libc.syscall(SYS_memfd_create, name.encode('utf8'), flags)
```

Comment:
I think we can use the libc wrapper:
import ctypes
libc = ctypes.CDLL(None)
return libc.memfd_create(name.encode('utf8'), flags)

In this case, we will not need the C program. What do you think about this?

Reply:
Yes, very well. Thank you.

I was assuming an old libc that doesn't have `memfd_create()`.
But for the tests, it's okay to assume a recent libc.

I pushed the changes.